src/internal/components/InternalEditor.tsx

```diff
@@ -151,7 +149,6 @@ export const InternalEditor = ({
         overrides={{
           header: () => {
             const { refreshPermissions, config } = usePuck();
-            setPuckRendered(true);
```

Comment:
I asked if there was a way to do this in Discord. They said we could try getting `iframeRef` from usePuck and then doing iframeRef.onload. Give that a shot to see if it allows us to remove the setTimeout

Reply:
Jk, it's not available yet. Someone just put up a PR for it https://github.com/measuredco/puck/pull/652

Reply:
You could try `#preview-frame` to access the iframe